src/Entity/RulesComponent.php

```diff
@@ -147,6 +156,64 @@ public function setExpression(ExpressionInterface $expression) {
   }
 
   /**
+   * Gets the configured component.
+   *
+   * @return \Drupal\rules\Engine\RulesComponent
+   *   The component.
+   */
+  public function getComponent() {
+    $component = \Drupal\rules\Engine\RulesComponent::create($this->getExpression());
```

Comment:
not easily possible as it's the name of the current class - let's leave this to the re-name follow-up